.github/workflows/deploy.yml

```diff
@@ -0,0 +1,144 @@
+                "env" : "dev",
+                "root_dns" : "cdn-dev.byu.edu",
+                "config_branch": "dev",
+                "certificate_arn" : "arn:aws:acm:us-east-1:632558792265:certificate/1bc2f81c-2f79-46b3-9d3b-54ce672ba8be"
+              }
+            ]
+          }'
+          echo matrix=`echo $matrix | jq -c .` >> $GITHUB_ENV
+
+      - name: Set up PRD Environment Variables
+        if: github.ref == 'refs/heads/master'
+        run: |
+          matrix='{
+            "env":[
+              {
+                "aws_key_name":"byu_oit_webcommunity_prd_key",
+                "aws_secret_name":"byu_oit_webcommunity_prd_secret",
+                "rfc_key_name":"standard_change_production_client_key",
+                "rfc_secret_name":"standard_change_production_client_secret",
+                "rfc_template_id":"Codepipeline-Standard-Change",
+                "cdn_name": "web-community-cdn",
+                "account_stack_name": "web-community-cdn-account",
+                "config_repo": "byu-oit/web-cdn",
+                "extra_tags": "data-sensitivity=public repo=https://github.com/byu-oit/web-cdn",
+                "env" : "prd",
+                "root_dns" : "cdn.byu.edu",
+                "config_branch": "master",
+                "certificate_arn" : "arn:aws:acm:us-east-1:204581410681:certificate/18c1b547-de51-43d3-afa5-2bd05493c41c"
+              }
+            ]
+          }'
+          echo matrix=`echo $matrix | jq -c .` >> $GITHUB_ENV
+
+    outputs:
+      matrix: ${{ env.matrix }}
+
+  build_and_deploy:
+    name: Build CDN pipeline
+    runs-on: ubuntu-latest
+    needs: env
+    strategy:
+      matrix: ${{ fromJson(needs.env.outputs.matrix) }}
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+
+      - name: Disallow Concurrent Runs
+        uses: byu-oit/github-action-disallow-concurrent-runs@v2
+        with:
+          token: ${{ github.token }}
+
+      - name: Configure AWS Credentials
+        id: awscreds
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets[matrix.env.aws_key_name] }}
+          aws-secret-access-key: ${{ secrets[matrix.env.aws_secret_name] }}
+          aws-region: us-east-1
+
+      - name: Setup
+        run: .codebuild/setup.sh
+
+      - name: Install AWSCLI
+        run: pip install awscli --upgrade --user
+
+      - name: Lerna Boostrap
+        run: lerna bootstrap
+
+      - name: Echo ROOT_DNS
+        env:
+          root_dns: ${{ matrix.env.root_dns }}
+        run: echo $root_dns
+
+      - name: Lerna Exec
+        env:
+          ENV: ${{ matrix.env.env }}
+          AWS_ACCOUNT_ID: ${{ steps.awscreds.outputs.aws-account-id }}
+          ROOT_DNS: ${{ matrix.env.root_dns }}
+        run: lerna exec -- ./codebuild.sh
+
+      - name: find
+        run: find ./* -mtime +10950 -exec touch {} \;
+
+#      - name: Start Standard Change
```

Comment:
At some point this needs to be added back

Reply:
Agreed, I added it back in.